src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -201,15 +201,17 @@ private IEnumerable<IConfigurationSection> GetFeatureDefinitionSections()
         {
             const string FeatureManagementSectionName = "FeatureManagement";
```

Comment:
ConfigurationFeatureDefitionProvider have these two property
private const string FeatureFiltersSectionName = "EnabledFor";
private const string RequirementTypeKeyword = "RequirementType";
It looks odd for me to have this const string in GetFeatureDefinitionSections().
I think this PR will be a good chance to change it into a const string property for the class. How do you think?

Reply:
This is a good point, but the preview branch actually has a change to use a ConfigurationFields.cs file with constants for each field as a result of the variants PR. That means this should no longer be an issue if we merge this into preview instead, like mentioned in the above comments.

Reply:
Now targeting preview and using the ConfiguartionFields file.